web/src/ui_util.ts

```diff
@@ -133,3 +133,16 @@ export function get_content_element($element: JQuery): JQuery {
     }
     return $element;
 }
+
+export function get_scroll_element($element: JQuery): JQuery {
+    const element = $element.expectOne()[0];
+    const sb = SimpleBar.instances.get(element);
+    if (sb) {
+        return $(sb.getScrollElement());
+    } else if ("simplebar" in element.dataset) {
+        // The SimpleBar mutation observer hasn’t processed this element yet.
+        // Create the SimpleBar early in case we need to add event listeners.
+        return $(new SimpleBar(element).getScrollElement()!);
+    }
+    return $element;
+}
```

Comment:
Hmm, looking at this more, I think it might be better to merge these into `web/src/scroll_util.js` rather than `ui_util.js` -- it's a more focused/specific module.

Reply:
Yeah `scroll_util` seems more appropriate module for them I will move them there.